exporter/containerimage/export.go

```diff
@@ -187,6 +190,14 @@ func (e *imageExporterInstance) ID() int {
 }
 
 func (e *imageExporterInstance) Name() string {
```

Comment:
Maybe `Type()` would be better?

Reply:
Potentially - note this also mirrors the structures of the cache exporter here.